wordpress-mobile/gutenberg-mobile
# Port KeyboardAvoidingView, KeyboardAwareFlatList and ReadableContentView to @wordpress/components

This PR is step 2 of our mobile refactor wordpress-mobile/gutenberg-mobile#958

Gutenberg PR: https://github.com/WordPress/gutenberg/pull/15913

In this PR we're moving `KeyboardAvoidingView`, `KeyboardAwareFlatList` and `ReadableContentView` from `src/components` to `gutenberg/packages/components/src/mobile`.
The goal is to have them available on `gutenberg` to be able to port other components there.

### Testing Instructions
- Check that tests passes
- Check that the app can load on iOS
- Check that the app can load on Android
- Check that there is no regression on the block list, the bottom sheet modal (block picker, image settings, adding link), or any editable block (Paragraph, Header, Image caption)
